src/main/resources/fleet_mgmt_location.avro

```diff
@@ -0,0 +1,56 @@
+{
+    "namespace": "fleet_mgmt",
+    "name": "fleet_mgmt_location",
+    "type": "record",
+    "fields": [
+        {
+            "name": "vehicle_id",
+            "type": {
+                "type": "int",
+                "arg.properties": {
+                    "range": {
+                        "min": 1000,
+                        "max": 9999
+                    }
+                }
+            }
+        },
+        {
+            "name": "latitude",
+            "type": {
+                "type": "double",
+                "arg.properties": {
+                    "range": {
+                        "min": -50,
+                        "max": 70
+                    }
+                }
+            }
+        },
+        {
+            "name": "longitude",
+            "type": {
+                "type": "double",
+                "arg.properties": {
+                    "range": {
+                        "min": -175,
+                        "max": 175
+                    }
+                }
+            }
+        },
+        {
+            "name": "timestamp",
```

Comment:
suggest naming it `ts` (or `unix_timestamp_ms` if we want to be really explicit) rather than `timestamp` since `timestamp` is a ksqlDB reserved word and since these would be used w/ ksqlDB. TBH I'm not sure if ksqlDB would be ok with a field named `timestamp` but even if it did I think it's cleaner / more readable if we avoid using keywords as field names